src/theme/index.tsx

```diff
@@ -42,55 +42,52 @@ export function colors(darkMode: boolean): Colors {
 
     // text
     text1: darkMode ? '#FFFFFF' : '#000000',
-    text2: darkMode ? '#DCDCDC' : '#565A69',
```

Comment:
Are all changes in this file reverted commits?

Reply:
I thnnk so, it's what I tried to do. Or u mean if I actually did a `git revert`?